plugins/package-managers/node/src/funTest/assets/projects/synthetic/pnpm/workspaces-expected-output.yml

```diff
@@ -50,16 +86,31 @@ analyzer:
       scopes:
       - name: "dependencies"
         dependencies:
-        - id: "NPM::chalk:4.0.0"
+        - id: "NPM::json-stable-stringify:1.0.1"
```

Comment:
Shouldn't `PNPM::pnpm-workspaces:1.0.1` also have `PROJECT_DYNAMIC` dependencies on the projects referred to by its workspces, i.e. on `PNPM::pnpm-app-example:1.1.4`, `PNPM::testing-pnpm-package-a:1.0.2` and `PNPM::testing-pnpm-package-b:1.0.2`?

Reply:
I'm actually not sure about this myself. Maybe a workspace declaration odes not really express a dependency relationship by itself, and thus these projects should not be listed here.

Reply:
I planned to investigate this in my following work. In pnpm I noticed that a workspace actually can have dependencies, but workspace submodules are not shown under dependencies. To me this makes sense. 
I believe a workspace is just some construct which helps with development, and also helps with using a single lockfile for all projects, but it does not really matter in terms of the actual dependencies. 

I propose to not add any such dependency from the workspace to the submodules. At least not in this PR. I guess there will be a better understanding on this on my side soon anyway, so if you're fine, let's defer the topic.